$NetBSD: patch-ab,v 1.7 2014/01/21 21:14:06 drochner Exp $

(from commit log)
avoid a hidden dependency on py-gtk2 which makes that an additional
file gets installed

--- setup.py.orig	2014-07-31 12:17:39.000000000 +0900
+++ setup.py	2014-09-01 22:40:41.000000000 +0900
@@ -94,11 +94,11 @@ mpl_packages = [
     setupext.BackendQt5(),
     setupext.BackendQt4(),
     setupext.BackendPySide(),
-    setupext.BackendGtk3Agg(),
-    setupext.BackendGtk3Cairo(),
-    setupext.BackendGtkAgg(),
-    setupext.BackendTkAgg(),
-    setupext.BackendWxAgg(),
+#   setupext.BackendGtk3Agg(),
+#   setupext.BackendGtk3Cairo(),
+#   setupext.BackendGtkAgg(),
+#   setupext.BackendTkAgg(),
+#   setupext.BackendWxAgg(),
     setupext.BackendGtk(),
     setupext.BackendAgg(),
     setupext.BackendCairo(),
